met-web/src/components/engagement/form/EngagementWidgets/Documents/FileDrawer.tsx

```diff
@@ -88,7 +115,14 @@ const FileDrawer = () => {
                         padding="2em"
                     >
                         <Grid item xs={12}>
-                            <MetHeader3 bold>Add File</MetHeader3>
```

Comment:
This is preferred when the condition is for a lot of code, but if it's a simple text perhaps it's just easier to go with your instincts and use ternary here like <MetHeader3> { !!documentToEdit ? 'Edit File' : 'Add File'} </MetHeader3>

Reply:
good idea